### ISSUE TITLE
Crashing upon load in ios 5.1.1

### ISSUE THREAD

**Issue by danielwill** [mattvv/lineplease#64]
I ran the app and started a script, everything was fine.

I noticed that autocorrect had renamed a character from Herr to Here, so I went in to edit it. Every time I tried to change the character name, the app crashed. But it would start again.

I went to the web site, the script was there, I continued to edit it. Then I went to the app and now it won't load at all. It crashes upon startup.

I deleted the app and downloaded it again. It now won't run without crashing--it seems to be trying to download the script from the web, then something happens and it crashes.

I deleted all scripts from the web interface, tried to load the app again and it still crashes, 

I'd love to use this, but so far can't get it to run reliably.

Suggestions?

**Comment by mattvv** [comment]
Hi daniel, fixes for both these issues are in 1.1.7 which was submitted to the app store last monday. You should see it approved and on your phone sometime this week! 

Thank you for your feedback!

**Comment by danielwill** [comment]
Thanks, Matt. I'll look forward to the update.

Part of the problem also seems to be a connection issue. When the iphone can't connect online, then it crashes. I've had wifi issues and it wouldn't connect, then it would crash. I turned off wifi and let it connect through the cellular network, and as long as it connected it would download the script and not crash. 

I also had issues editing the script online. When I tried to delete an "empty" line (just the character name, no dialog), I repeatedly got an error from the web site.
